src/client.ts

```diff
@@ -10,12 +12,14 @@ import type {
   Thread,
   UserFeedback,
 } from '@sentry/types';
-import { logger, SentryError } from '@sentry/utils';
+import { dateTimestampInSeconds, logger, SentryError } from '@sentry/utils';
 
 import type { CapacitorClientOptions } from './options';
 import { mergeOutcomes } from './utils/outcome';
 import { NATIVE } from './wrapper';
 
+export const EnvelopeitemsIndex = 1;
```

Comment:
L: I would not export this if used only in the client file.

M: Missing capital I.
```suggestion
export const EnvelopeItemsIndex = 1;
```